Makefile

```diff
@@ -73,6 +73,9 @@ docker-image-os-coreos:
 docker-image-os-coreos-alicloud:
 	@docker build --build-arg VERIFY=$(VERIFY) -t $(IMAGE_PREFIX)/gardener-extension-os-coreos-alicloud:$(VERSION) -t $(IMAGE_PREFIX)/gardener-extension-os-coreos-alicloud:$(VERSION) -f Dockerfile --target gardener-extension-os-coreos-alicloud .
 
+.PHONY: docker-image-provider-aws
+docker-image-provider-aws:
+	@docker build --build-arg VERIFY=$(VERIFY) -t $(IMAGE_PREFIX)/gardener-extension-provider-aws:$(VERSION) -t $(IMAGE_PREFIX)/gardener-extension-provider-aws:$(VERSION) -f Dockerfile --target gardener-extension-provider-aws .
```

Comment:
Can you add it to the [`Dockerfile`](https://github.com/gardener/gardener-extensions/blob/feature/infrastructure/Dockerfile)?